custom/icds_reports/tasks.py

```diff
@@ -1209,6 +1249,114 @@ def build_incentive_report(agg_date=None):
                 build_incentive_files.delay(location, agg_date, file_format, 3, location.parent.parent, location.parent)
 
 
+# performance row is AWCIncentiveReport object with 'awc_id', 'is_launched', 'valid_visits',
+# 'visit_denominator', 'wer_weighed', 'wer_eligible', 'incentive_eligible'
+# awcagg_row is AggAwc object with 'awc_id', 'is_launched', 'valid_visits', 'expected_visits'
+def get_awc_is_launched_mismatch_row(performance_row, awcagg_row):
```

Comment:
These might be clearer if you use `values` instead of `values_list` because then the dict keys will explain what you are accessing.